src/connectors/autocomplete/__tests__/connectAutocomplete-test.ts

```diff
@@ -859,11 +859,10 @@ search.addWidgets([
         1
       );
 
-      instantSearchInstance.mainHelper!.hasPendingRequests = () => true;
-      instantSearchInstance._isSearchStalled = true;
-      instantSearchInstance.scheduleRender();
-
-      await wait(0);
+      // this client never resolves, thus search is stalled
+      instantSearchInstance.client.search = () => new Promise(() => {});
+      instantSearchInstance.scheduleSearch();
+      await wait(400);
```

Comment:
Are these fake timers? Otherwise we can decrease this amount with `stalledSearchDelay`.

Reply:
had to add a margin, probably due to timers being "minimal", and them allowing to overtake eachother